Microsoft.Toolkit.Win32/Microsoft.Toolkit.Forms.UI.Controls/Microsoft.Toolkit.Forms.UI.Controls.csproj

```diff
@@ -0,0 +1,64 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net462</TargetFramework>
+    <RootNamespace>Microsoft.Toolkit.Forms.UI.Controls</RootNamespace>
+    <AssemblyName>Microsoft.Toolkit.Forms.UI.Controls</AssemblyName>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <RunCodeAnalysis>true</RunCodeAnalysis>
+
+    <Title>Windows Community Toolkit Windows Forms Controls</Title>
+    <Description>This library provides XAML user controls for Windows Forms. It is part of the Windows Community Toolkit.</Description>
+    <PackageTags>WebView Win32 WindowsForms</PackageTags>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DefineConstants>$(DefineConstants);DEV_DEBUG;DEBUG_LAYOUT</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\Microsoft.Toolkit.Win32.UI.Controls\**\*.cs" LinkBase="Shared" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="PresentationCore" />
+    <Reference Include="PresentationFramework" />
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Design" />
+    <Reference Include="System.Drawing" />
+    <Reference Include="System.Windows.Forms" />
+    <Reference Include="System.Xaml" />
+    <Reference Include="WindowsBase" />
+    <PackageReference Include="System.Net.Http" Version="4.0.0" />
+    <PackageReference Include="System.Runtime.WindowsRuntime" PrivateAssets="All">
```

Comment:
Oren had issue with this previously. The version that ships with .NET isn't directly resolvable and was switched to a `Reference` with a `Hintpath`

Reply:
We also should not take a PackageReference to `System.Net.Http` for .NET FX projects. That should be a regular `Reference`.